Alignment/APEEstimation/plugins/ApeEstimator.cc

```diff
@@ -129,7 +129,7 @@
 class ApeEstimator : public edm::EDAnalyzer {
```

Comment:
@davidlange6 - this should be thread-safe analyzer. The rest of API changes would follow.